src/serve/routes/addresses/all_rune_balances.rs

```diff
@@ -82,13 +89,43 @@ pub async fn addresses_all_rune_balances(
         }
     }
 
-    let mut processed_balances = vec![];
+    let mut processed_balances: Vec<RuneBalanceWithInfo> = vec![];
 
     for (rune_id, amount) in balances.into_iter().sorted_by_key(|(rid, _)| *rid) {
-        processed_balances.push(RuneAndAmount {
+        let rune_info = if params.include_info {
+            match storage.get_maybe::<RuneInfoByIdKV>(&rune_id)? {
```

Comment:
If we have balance for a rune, then it follows that we must have also processed the deployment, and so this KV should exist, and we can use .get_expected() instead which will return an error if not found